dds_web/__init__.py

```diff
@@ -185,6 +185,20 @@ def create_app(testing=False, database_uri=None):
             # Simplifies testing as we don't test the session protection anyway
             login_manager.session_protection = "basic"
 
+        @app.after_request
```

Comment:
We want to check the version before the request and this checks it after the request has been made and the API is returning a response.

Reply:
So in this case something could be executed even though it shouldn't be because the version is incorrect